src/collections/_documentation/workflow/integrations/gitlab.md

```diff
@@ -0,0 +1,50 @@
+---
+title: GitLab
+sidebar_order: 6
+---
+
+Sentry’s new GitLab integration helps you find and fix bugs faster by using data from your GitLab commits. Additionally, you can streamline your triaging process by creating a GitLab issue directly from Sentry.
+
```

Comment:
One last thing I forgot -- can we add something here that says:

`Note: this feature is currently available for Early Adopters. You can become an Early Adopter by navigating to your organization settings > General Settings.`